### ISSUE TITLE
Provide classes that result in nodes with more reasonable names

### ISSUE THREAD

**Issue by jonnew** [jonnew/Bonsai.ONIX#4]
When helping people with remote installations, the names of nodes used in Bonsai workflows are a source of confusion. For instance, even though a low-level device description might be "FMC Headstage Control Device", there is no reason we to call a corresponding class `FMCHeadstageControlDevice`. FMC has nothing to do with this. Another example: `BNO055` instead of something that describes its function. I would like to brainstorm name replacements below and come up with some sort of unified scheme.

**Comment by aacuevas** [comment]
We want to still be able to add new devices that share a functionality (for example different IMUs) so device name should probably part of the class names, maybe something like `BNO055IntertialMeasurementUnit` .

**Comment by jonnew** [comment]
Yes, agree on that. Honestly, an issue here is that the Bonsai workflow editor derives node display names from class names without namespace etc.  So we are going to have some long names, like the one you suggest for instance, which I don't love. But the alternative is bad too -- having a node named IMU will of course conflict with something in the future.
